environment/vagrant/training/vb-multi-vm/TestIndyClusterSetup.md

```diff
@@ -1,14 +1,16 @@
 # Setting Up a Test Indy Network in VMs
 
+**WARNING:** Here use deprecated client. Please, use [new client from SDK](https://github.com/hyperledger/indy-sdk/tree/master/cli)
+
 When you're finished working through this document, you will be able to proceed through to the [*Getting
-Started Guide*](../../../../getting-started.md) or if you would like, you may continue setting up an actual Developer Environment connected to a sandbox by following these [instructions](https://github.com/hyperledger/indy-node/blob/master/environment/vagrant/sandbox/DevelopmentEnvironment/Virtualbox/Vagrantfile).
+Started Guide*](https://github.com/hyperledger/indy-sdk/blob/master/doc/getting-started/run-getting-started.md) or if you would like, you may continue setting up an actual Developer Environment connected to a sandbox by following these [instructions](https://github.com/hyperledger/indy-node/blob/master/environment/vagrant/sandbox/DevelopmentEnvironment/Virtualbox/Vagrantfile).
```

Comment:
I don't think this is a correct statement. This guide is for the old GSG, not the new one.

Reply:
Will back to the old link. Are here should be link to a new GSG or warning about deprecated this GSG?

Reply:
Or warning with a new link in the start willl be enough?